### PR TITLE
Technical debt - pep8 compliance

### PR DESCRIPTION
**Pull request by Sybrand**
- Changed json structure to comply with python conventions.
- Changed code to comply to linting rules.
- Removed data for stations not used.
- Changed how Dockerfile works (running in production mode, producing "clean" image).
- Added Dockerfile-lint for testing linting.
- Updated docker-compose 
- Updated /stations/ endpoint to be REST compliant.
- Removed ENABLE_PIPENV environment variable to ensure consistent build on local machine and in openshift.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -18,8 +18,8 @@ pylint = "*"
 [packages]
 fastapi = "*"
 uvicorn = "*"
-python-multipart = "*"
 requests = "*"
+gunicorn = "*"
```
Comment (Kyubinhan) [inline Pipfile]: Do we need another package to run the server? I thought unicorn can handle that

File: Dockerfile [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
+RUN python -m pip install -r /tmp/requirements.txt
+
+WORKDIR /app
 COPY . .
+
 EXPOSE 8080
-CMD uvicorn main:app --reload --host 0.0.0.0 --port 8080
\ No newline at end of file
+
+CMD gunicorn main:APP -w 4 -k uvicorn.workers.UvicornWorker -b :8080
```
Comment (Kyubinhan) [inline Dockerfile]: I feel we need another Dockerfile where it reloads when the file system changes.

File: Dockerfile-lint [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# This dockerfile gives you a container with pylint, a development package installed.
```
Comment (Kyubinhan) [inline Dockerfile-lint]: Can you rename this file to lint.Dockerfile?
https://stackoverflow.com/a/26084010

File: Pipfile (hunk 1 above)
Comment (Sybrand) [reply Pipfile]: See: https://www.uvicorn.org/#running-with-gunicorn

uvicorn gives us a far server (async etc.)
gunicorn gives us a strong server (managing threads & processes)

So gunicorn and uvicorn work together to give us something fast and stable.

File: Dockerfile (hunk 2 above)
Comment (Sybrand) [reply Dockerfile]: You can override the cmd when you're doing development. Look at the Makefile,
`docker-compose run api uvicorn main:APP --reload --host 0.0.0.0 --port 8080`
So I don't think there's a need to create an entire new dockerfile. `docker-compose up` will give you a production style, but you can override that.

Comment (Kyubinhan) [reply Dockerfile]: Gotcha!

File: Pipfile (hunk 1 above)
Comment (Kyubinhan) [reply Pipfile]: 👍

**Pull request merged**